ivy/functional/backends/tensorflow/activations.py

```diff
@@ -46,6 +46,12 @@ def relu(x: Tensor, /, *, complex_mode="jax", out: Optional[Tensor] = None) -> T
     return tf.nn.relu(x)
 
 
+def segment_max(
+    x: Tensor, /, *, complex_mode="jax", out: Optional[Tensor] = None
```

Comment:
the argument `segment_ids` shouldnt be omitted

Reply:
I have added `segment_ids`